.gitignore

```diff
@@ -16,3 +16,6 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots/**/*.png
 fastlane/test_output
+
+# access keys
+Client/Keys.swift
```

Comment:
Is it a file to hide the secret keys? Would be good to add something to the Readme file

Reply:
do we need to update .gitignore?